aws/aws-how-to/instances/build-pro-ami-using-packer.rst

```diff
@@ -9,7 +9,7 @@ We'll be using Ubuntu Pro 20.04 for this guide, but the method is equally applic
 
    * For **Ubuntu Pro FIPS**, it is better to use a pre-enabled FIPS image from the Marketplace to avoid unnecessary additional steps. 
    * For **Ubuntu LTS**, you can use this method with a small change as explained at the end of the :ref:`define-provisioners` section below. 
-
+   * For **Ubuntu 22.04 LTS and above** you need packer version 1.8.1 or newer.
```

Comment:
later, you write `1.7.1` is required. which version is it?

Reply:
Good catch! it is 1.8.1. Let me update the docs.